### PR TITLE
DO NOT MERGE - WAVE2-709 - content ownership - fix for new content

### PR DESCRIPTION
**Pull request by balrajb**
This PR just for reference until we merge/test https://github.com/ucalgary/www.ucalgary/pull/368

### CODE REVIEW

File: sites/all/modules/ucalgary/content_ownership/content_ownership.admin.inc [hunk 1]
```diff
@@ -47,30 +47,34 @@ function content_ownership_add_form_group_ownership_options(array &$form, &$form
     }
     else {
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership.admin.inc]: More comments here on what these conditions are and what they are doing.

File: sites/all/modules/ucalgary/content_ownership/content_ownership.admin.inc [hunk 2]
```diff
@@ -160,18 +165,19 @@ function content_ownership_add_form_group_ownership_options(array &$form, &$form
...
+function _update_content_ownership_table($entity_id, $type, $subtype, $ownership_groups, $entity_machine_name = NULL) {
     if ($type == 'entity' && $subtype == 'menu' && isset($entity_machine_name)
-        && isset($entity_id) && (int)$entity_id == 0) {
-        $entity_id = null;
+        && isset($entity_id) && (int) $entity_id == 0
+    ) {
+        $entity_id = NULL;
     }
 
     $query = db_select('content_ownership', 'co');
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/content_ownership/content_ownership.admin.inc]: This block of code is nearly identical to the one found on line 58. Could they be merge into a single function that can be used as a callback?

**Comment by dpfitzsi** [comment]
Balraj can you fix these conflicts?

**Comment by dpfitzsi** [comment]
No longer relevant under new structure

**Comment by balrajb** [comment]
@dpfitzsi sure, go for it!

**Pull request closed without merging**